packages/core/src/import-models.ts

```diff
@@ -35,6 +35,9 @@ export async function importModels(globPaths: string | string[], modelMatch?: Mo
 
 async function importModelNoGlob(url: string, modelMatch?: ModelMatch): Promise<ModelStatic[]> {
   const module = await import(url);
+  if (module.default && !isModelStatic(module.default) && typeof module.default === 'object') {
```

Comment:
Adding some info for future maintainers

```suggestion
  // when importing a CJS file, sometimes only the default export is available, as named exports depend on the file's exports being statically analyzable by node. The default exports contains the contents of the file's `module.exports`
  if (module.default && !isModelStatic(module.default) && typeof module.default === 'object') {
```